Podfile

```diff
@@ -7,6 +7,8 @@ use_frameworks!
 workspace 'AEPEdge'
 project 'AEPEdge.xcodeproj'
 
+pod 'SwiftLint', '0.44.0'
```

Comment:
For this one I think we also need to update the scripts for the xcode project Build Phases -> Script Phase